designer/server/src/routes/forms/api.js

```diff
@@ -33,16 +33,17 @@ export default [
         parse: true
       },
       async handler(request, h) {
-        const { id } = request.params
+        const { auth, params, payload } = request
+        const { id } = params
+        const author = forms.getAuthor(auth.credentials)
+
         try {
-          const { value, error } = formDefinitionSchema.validate(
-            request.payload,
-            {
-              abortEarly: false
-            }
-          )
+          const result = formDefinitionSchema.validate(payload, {
```

Comment:
Thanks for deferring the destructuring to fix that ESLint warning 🙌

Reply:
Kinda wishing I hadn't bothered! :rofl: - it takes longer to then discuss code styles!

Reply:
Nah it's all worth it and this feedback is good, otherwise we'll each do things 3x different ways!

Good to leave things in order for the next person